extensions/grpc/deployment/src/test/java/io/quarkus/grpc/client/MutinyClientInjectionTest.java

```diff
@@ -35,6 +41,12 @@
     public void test() {
         String neo = service.invoke("neo-mutiny");
         assertThat(neo).matches("Hello neo-mutiny");
+
```

Comment:
it would be good to also test if client interceptors are invoked with the same duplicated context

Reply:
The test checks that it does call you on the same duplicated context. Not the interceptor, but the actual items.

Reply:
it seems we didn't understand each other :)
OTOH, it's unlikely interceptor invocation would hijack context somehow, so maybe we're good without a test I wrote a bout